.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Feature Request
+description: Suggest an idea for this project
+title: "[Feature]: "
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
+  - type: textarea
+    id: problem-statement
+    attributes:
+      label: What prompted this feature request?
+      description: Is your feature request related to a problem? Please describe.
+      placeholder: I get frustrated every time <x> happens, it interrupts my workflow
```

Comment:
I suggested @morganzellers reword this on the form in the schema repo, but in this case I like the wording. And now I think maybe we should use this there as well. 🤔

Reply:
I'll update the Workflow Schema PR with this text